alacritty_terminal/src/term/mod.rs

```diff
@@ -1865,18 +1865,19 @@ impl<T: EventListener> Handler for Term<T> {
                     cell.reset(&template);
                 }
 
+                let last_visible_line_index = num_lines - 1;
```

Comment:
The variable name and definition is longer than the variable itself. What's the point?